### PR TITLE
Code formatting

### PR DESCRIPTION
**Pull request by yanlyub**
Fixes a bunch of checkstyle warnings.

### CODE REVIEW

File: vassal-app/src/main/java/VASSAL/build/module/Inventory.java [hunk 1]
```diff
@@ -341,8 +341,8 @@ public void paintIcon(Component c, Graphics g, int x, int y) {
 
                 piece.draw(
                   g,
-                  (int)(-r.x * os_scale),
-                  (int)(-r.y * os_scale),
+                  (int) (-r.x * os_scale),
+                  (int) (-r.y * os_scale),
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/build/module/Inventory.java:345]: I hadn't realized we'd enabled this one. Could we turn this back off?

File: vassal-app/src/main/java/VASSAL/build/module/map/SetupStack.java [hunk 2]
```diff
@@ -231,12 +231,12 @@ public void setup(boolean gameStarting) {
         p.translate(r.x, r.y);
       }
       if (placeNonStackingSeparately()) {
-        for (int i=0;i<s.getPieceCount();++i) {
+        for (int i=0; i<s.getPieceCount(); ++i) {
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/build/module/map/SetupStack.java:234]: Spacing around `=`, `<` here.

**Comment by uckelman** [comment]
Otherwise looks ok.

File: vassal-app/src/main/java/VASSAL/build/module/map/SetupStack.java (hunk 2 above)
Comment (yanlyub) [reply vassal-app/src/main/java/VASSAL/build/module/map/SetupStack.java:234]: Will be checked by rule "WhitespaceAround" that we will enable later on. For now we have enabled "WhitespaceAfter".

**Pull request merged**